pkg/fqdn/dnsproxy/udp.go

```diff
@@ -42,6 +42,11 @@ var udpOOBSize = func() int {
 type sessionUDPFactory struct {
 	// A pool for UDP message buffers.
 	udpPool sync.Pool
+
+	// ipv4Enabled and ipv6Enabled are used when setting up the proxy sockets
+	// later, and determine if we bind to 127.0.0.1 and ::1, respectively.
+	// See sessionUDPFactory.SetSocketOptions
+	ipv4Enabled, ipv6Enabled bool
```

Comment:
Looks like these would need to be named `ipv4` and `ipv6` for the code to compile?

Reply:
😊 yup... a few things were wrong :/